PopPUNK/bgmm.py

```diff
@@ -45,10 +45,36 @@ def fit2dMultiGaussian(X, dpgmm_max_K = 2):
     return dpgmm
 
 
+def findBetweenLabel_bgmm(means, assignments, rank = 0):
+    """Identify between-strain links
+
+    Finds the component with the largest number of points
+    assigned to it
+
+    Args:
+        means (numpy.array)
+            K x 2 array of mixture component means
+        assignments (numpy.array)
+            Sample cluster assignments
+        rank (int)
+            Which label to find, ordered by distance from origin. 0-indexed.
+            (default = 0)
```

Comment:
I think ordered by number of assigned samples, not distance from the origin?

Reply:
Pointless. Removed in 6bbfdb7.